packages/codemirror-extension/src/index.ts

```diff
@@ -404,7 +404,7 @@ function activateEditorCommands(
     ['sublime', trans.__('sublime')],
     ['vim', trans.__('vim')],
     ['emacs', trans.__('emacs')]
-  ].forEach(([name, displayName])=> {
+  ].forEach(([name, displayName]) => {
```

Comment:
Also fixed in another PR I merged just now. (And sorry, this came from one of my PRs)